pulsar-io/kinesis/pom.xml

```diff
@@ -84,6 +84,12 @@
       <version>${avro.version}</version>
     </dependency>
 
+    <dependency>
+      <groupId>com.github.wnameless.json</groupId>
+      <artifactId>json-flattener</artifactId>
+      <version>0.13.0</version>
```

Comment:
it's better to define a version instead of hardcode version here. like avro

Reply:
A lot of deps in this pom don't use versions in properties (eg. amazon-kinesis-client)... but ok...

Reply:
Done.
@Shoothzj can you give another look ?